### DISCUSSION TITLE
Welcome to Klipper-Configs Discussion for the Ender3 v2 with Sprite Pro Extruder and a 4.2.7 board!

### DISCUSSION THREAD

**Discussion by LeeOtts** [LeeOtts/Ender3v2-Klipper-Configs#1] [Announcements]
Thanks for stopping by, I just updated my modified Ender 3 v2 to Klipper. Due to modifications such as the Sprite Pro Extruder and 4.2.7 Silent board, there were no premade files. Feel free to use this as a blueprint for your Klipper printer.conf

**Comment by placingpixels** [comment]
Hey thanks for sharing this! My old e3 is screaming.

**Reply by LeeOtts** [reply]
Cool beans. Did some tweaking and adding KAMP to it this week. Happy Printing!

**Comment by Chriz2688** [comment]
Hello, can I also use the config for the Ender 3 Pro?

I have an E3 Pro with a Sonic Pad, 4.2.7 Board with Klipper. Yesterday I installed the sprite extruder but the CR Touch stops when autoleveling. He runs out of the bed.

Thank you

**Reply by LeeOtts** [reply]
Hey, sorry I am just now seeing this. It has been a busy couple of weeks. 

It should work with Ender 3 Pro.  The board pins are the same as the 4.2.7 board and the size of the machine is the same.

If you hit the sides, you may need to tweak 

[bedmesh] 
mesh_max 

 [stepper_y]
position_max:
position_min: 

 [stepper_x] 
position_max:
position_min: 

I am not sure what you can add with the sonic pad

**Reply by ElTorro5150** [reply]
I’m sorry I’m just now replying as well. I have an Ender 3 MAX with a 4.2.7 board, sprite, CR Touch, and a sonic pad. Lee’s config file worked fine, I had to change the bed size and make some other tweaks to adjust for the higher Z and bed screw locations, you should NOT have to do any of that because your bed is the same size as an Ender V2. 

Navigate to Moonraker via a browser window using your SonicPad’s IP address on your network and upload the printer.cfg file through Moonraker. Make sure to save your current printer.cfg first! Just rename it to printer.old, then upload Lee’s file.
